File: 020-serpro-estudos-prova-pratica/src/especificacao-sem-bd/nivel-avancado/Tarefa.py
class Tarefa:
	def __init__(self, titulo):
		self.titulo = titulo
		self.concluida = False

	def descricao(self):
		retorno = "["
		if self.concluida:
			retorno += 'x'
		else:
			retorno += ' '
		retorno += "]"
		retorno += f' {self.titulo}'
		return retorno

	def concluir(self):
		self.concluida = True

File: 020-serpro-estudos-prova-pratica/src/especificacao-sem-bd/nivel-medio/Tarefa.java
public class Tarefa {
	private String titulo;
	private boolean concluida;

	public Tarefa(String titulo) {
		this.titulo = titulo;
		this.concluida = false;
	}

	public String titulo() {
		return this.titulo;
	}

	public boolean concluida() {
		return this.concluida;
	}

	public void concluir() {
		this.concluida = true;
	}

	@Override
	public String toString() {
		return "[titulo=" + titulo + ", concluida=" + concluida + "]";
	}
}